export/index.template.html

```diff
@@ -399,12 +399,67 @@ <h2>Message log</h2>
         delete jwkPrivateCopy.d;
         jwkPrivateCopy.key_ops = ["verify"];
 
-        const publicKey =  await window.crypto.subtle
-          .importKey("jwk", jwkPrivateCopy, { name: "ECDSA", namedCurve: "P-256" }, true, ["verify"]);
+        const publicKey =  await window.crypto.subtle.importKey(
+          "jwk", 
+          {
+              kty: jwkPrivateCopy.kty,
+              crv: jwkPrivateCopy.crv,
+              d: padBase64Url(jwkPrivateCopy.d, 32),
+              x: padBase64Url(jwkPrivateCopy.x, 32),
+              y: padBase64Url(jwkPrivateCopy.y, 32),
+              ext: jwkPrivateCopy.ext,
+          }, 
```

Comment:
I'm not sure this is correct: we specifically delete `d` here so that it's not part of the jwk. I also don't think there's a need to pad because we start with a valid JWK (the private key) and simply delete the `d` component to re-export a verifying public key which doesn't have the private information (`d`).

Reply:
good catch with the d field. Yeah we found testing locally that there wasn't variance in the key length -- discussed with @andrewkmin that it might be good to push anyway.

Reply:
I don't know that it's necessary actually: by definition the values in the JWK are valid, because they're already in the JWK. There's no need to export them, pad them "just to make sure" they're importable since they're part of the input JWK to start with (remember, the error we're seeing is an error to import buffers or numbers into a JWK format!)

The reason why I'm pushing back is the amount of new code is decently large.

Reply:
Yes yes yes

Reply:
@r-n-o makes sense! thanks for the catch here -- I'll go ahead and close the PR.

Reply:
No worries! If you wanted you could keep this PR opened and merge the extra test you have. But no big deal either way!